aurelia-slickgrid/src/aurelia-slickgrid/filters/compoundDateFilter.ts

```diff
@@ -211,15 +211,6 @@ export class CompoundDateFilter implements Filter {
     return $filterContainerElm;
   }
 
-  private getCurrentLocale(columnDef: Column, gridOptions: GridOption) {
-    const params = gridOptions.params || columnDef.params || {};
-    if (params.i18n && params.i18n instanceof I18N) {
-      return params.i18n.getLocale();
-    }
-
-    return 'en';
```

Comment:
you removed this function, but what happen if the user didn't do a full setup `i18n`? I can't remember what is the minimal even if you have only 1 locale to support, I think that I required to at least setup `i18n` with 1 locale. You might be in a better situation to confirm since you use only 1 locale, right? I just want to make sure that there will always be a locale returned on line 100

Reply:
oh yeah I meant to ask you about that. I guess when I tested it I forgot that you probably set it up in your demo? I never used this library so I thought that since it returned "en" whenI removed that function that the library is smart enough to return the default language

Reply:
if you aren't using `i18n` then what does that return? If nothing is found, it should always return 'en'

Reply:
yeah, if it i18n is not setup in main.ts, the currentLocale will return undefined. the new commit should fix it